app/jobs/v3/create_binding_async_job.rb

```diff
@@ -87,6 +92,19 @@ def handle_timeout
         )
       end
 
+      def save_failure(error_message)
```

Comment:
The resource can be updated by the action or the job, which I think is confusing. I think it would be better if all the updating of the resource were handled in the action.

Reply:
I agree- will put a refactoring in the backlog to see how the updates of the bindings (here and in handle timeout) can be removed from the job

https://www.pivotaltracker.com/story/show/175283473